cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/admin-role-dbms.asciidoc

```diff
@@ -1,19 +0,0 @@
-As <<administration-security-administration-introduction, described above>>, the `admin` role has a number of built-in privileges.
```

Comment:
Content was moved to cypherManual/asciidoc/access-control/dbms-administration.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/dbms-administration/